apache/superset
# fix: feature flags typing

### SUMMARY

Adds type annotations to feature flags, clearly stating that all feature flags are booleans. Because of that `QUERY_COST_FORMATTERS_BY_ENGINE`, `SCHEDULED_QUERIES` and `SQL_VALIDATORS_BY_ENGINE` are no longer feature flags but config keys. Note that currently on master, `SQL_VALIDATORS_BY_ENGINE` is defined as a top level key on `config.py`, but is erroneously being read from the feature flag object on the frontend. This fixes the issue and adds a note to `UPDATING.md`.

### Testing instructions
1. Go to SQL Lab and open a Postgres database
2. Write a query that's incorrect and verify that the error shows up in the gutter (see below)

![image](https://user-images.githubusercontent.com/33317356/142435963-903d94a4-b988-4552-9f4c-a6f09657804b.png)


### ADDITIONAL INFORMATION
